test/index.spec.ts

```diff
@@ -26,12 +27,34 @@ describe('IlcSdk', () => {
     describe('constructor options', () => {
         it('should correctly set default options', () => {
             const ilcSdk = new IlcSdk();
-            expect((<any>ilcSdk).log).equal(console);
-            expect((<any>ilcSdk).defaultPublicPath).equal('/');
+            expect((ilcSdk as any).log).equal(console);
+            expect((ilcSdk as any).defaultPublicPath).equal('/');
         });
     });
 
     describe('processRequest', () => {
+        describe('getCurrentReqHost', () => {
+            it('should parse request host correctly', () => {
+                const req = new MockReq(merge({}, defReq));
+                const res = ilcSdk.processRequest(req);
+
+                expect(res.getCurrentReqHost()).to.eq(defReq.headers['x-request-host']);
+            });
+
+            it('should fallback to dumb request host if not passed', () => {
+                const reqConf = merge({}, defReq);
+                delete reqConf.headers['x-request-host'];
```

Comment:
why don't we return `host` header in this case?

Reply:
I thought that  `host` header may be an unpredictable value... Let's discuss...